cogs/currency.py

```diff
@@ -0,0 +1,34 @@
+from discord import slash_command
+from discord.ext import commands
+import utils
+from utils import *
```

Comment:
I would suggest you simply do `import utils` in order to not clutter the namespace by doing `from utils import *`